kernel/run.py

```diff
@@ -372,23 +382,19 @@ def pmem_paths(args):
                     paths.append(default)
                     pmem = "on"
                 else:
-                    if len(paths) > 0:
-                        paths.append(paths[node % num_host_numa_nodes])
-                    else:
-                        f = open("{}".format(pmem_test_path), "w+")
-                        paths += [pmem_test_path] * args.qemu_nodes
-                        return paths
+                    path = "{}-{}".format(pmem_test_path, node)
+                    open(path, "w+")
+                    paths.append(path)
         return paths
 
     host_numa_nodes_list = query_host_numa()
     num_host_numa_nodes = len(host_numa_nodes_list)
     if args.qemu_nodes and args.qemu_nodes > 0 and args.qemu_cores > 1:
-        if int(args.qemu_pmem):
-            pmem_paths = pmem_paths(args)
-            assert len(pmem_paths) == args.qemu_nodes
+        if args.qemu_pmem and int(args.qemu_pmem):
```

Comment:
can we avoid this if we do `type=int` in the parser?

should probably do the same for qemu-memory then?
```
parser.add_argument("--qemu-memory", type=int,
                    help="How much total memory in MiB (will get evenly divided among nodes).", default=1024)
parser.add_argument("--qemu-pmem", type=int,
                    help="How much total peristent memory in MiB (will get evenly divided among nodes).", required=False, default=None)
```

Reply:
Done.